apps/src/lib/kits/maker/ui/SetupInstructions.jsx

```diff
@@ -97,6 +114,79 @@ class ConnectionInstructions extends React.Component {
   }
 }
 
+class CPExpressInstallInstructions extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    // TODO: layout with images
+    // TODO: get screenshot for image copy
+    // TODO: reset button gif- put in our cdn?
+    // TODO: make i18n strings for all the strings
+    return (
+      <div id="CPExpressInstallInstructions">
+        <h2>Install the Firmata firmware onto the CP Express</h2>
+        <ol>
+          <li>
+            Set your Circuit Playground Express to Bootloader Mode
+            <div style={style.oneColumn}>
+              <ul>
+                <li>
+                  Press or double-press the RESET button in the center of the
```

Comment:
@mgc1194 - just curious to see if you think we should leave 'RESET' out of the I18n string or not.

Reply:
Decided on yes!